pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -36,6 +36,7 @@
         "tempfile.NamedTemporaryFile",
         "tempfile.SpooledTemporaryFile",
         "tempfile.TemporaryDirectory",
+        "tempfile.TemporaryFile",
```

Comment:
This is an unrelated change right? Or am I missing something? If so, could you please make another PR for this?

Reply:
Yes, I can remove.

Reply:
Moved to #5468.